docs/configuration-reference/components/contour.md

```diff
@@ -60,6 +64,7 @@ Table of all the arguments accepted by the component.
 | `node_affinity`     | Node affinity for deploying the operator pod and envoy daemonset.                                       |       -        | list(object({key = string, operator = string, values = list(string)}))                                         |  false   |
 | `service_type`      | The type of Kubernetes service used to expose Envoy. Set as "NodePort" on the **AWS** platform.         | "LoadBalancer" | string                                                                                                         |  false   |
 | `toleration`        | Tolerations that the operator and envoy pods will tolerate.                                             |       -        | list(object({key = string, effect = string, operator = string, value = string, toleration_seconds = string })) |  false   |
+| `envoy.metrics_scrape_interval` | Interval at which Prometheus will scrape Envoy. Valid only when `enable_monitoring` is set to `true`. |       -        | string                                                                                               |  false   |
```

Comment:
Table is not aligned + we should validate this document condition and have unit tests for it.